uvm/impl/com/untangle/uvm/UvmContextImpl.java

```diff
@@ -1740,9 +1742,23 @@ private void createUID()
         if (uidFile.exists() && uidFile.length() > 0)
             return;
 
-        // if its devel env just return
-        if (isDevel())
-            return;
+        // if its devel env use existing uid if possible, otherwise create one
+        if (isDevel()) {
+            File coreUidFile = new File(CORE_UID_FILE);
+            boolean copySuccess = true;
+            if (coreUidFile.exists()) {
+                try{
+                    IOUtil.copyFile(coreUidFile, uidFile);
+                }catch(Exception e){
+                    logger.error("Unable to copy " + e);
+                    copySuccess = false;
+                }
+                if (copySuccess) {
+                    return;
```

Comment:
if createUID fails here we are returning, continue and try to generate a UID with the create UID script